app/views/topical_events/show.html.erb

```diff
@@ -51,75 +61,107 @@
-      <%= render(partial: 'shared/document_list_from_rummager', locals: { documents: @announcements['results'], type: :announcements, documents_count: @announcements['total'] }) if @announcements['results'].any? %>
-
-      <% if @detailed_guides.any? %>
-        <section id="detailed-guidance" class="document-block documents-<%= document_block_counter %>">
-          <h1 class="label">Detailed guidance</h1>
-          <div class="content">
-            <ol class="document-list">
-              <% @detailed_guides.each do |detailed_guide| %>
-                <%= content_tag_for :li, detailed_guide, class: 'document-row' do %>
-                  <h2><%= link_to detailed_guide.title, public_document_path(detailed_guide), class: "govuk-link" %></h2>
-                <% end %>
-              <% end %>
-            </ol>
-          </div>
-        </section>
-      <% end %>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <%= render partial: 'shared/recently_updated', locals: { 
+          recently_updated: @recently_changed_documents,
+          atom_url: atom_feed_url_for(@topical_event),
+          see_all_link: latest_path(topical_events: [@topical_event]),
+          documents_source: 'rummager'
+        } %>
+      </div>
+    </div>
 
-      <% if @travel_advice.any? %>
-        <section id="travel-advice" class="document-block documents-<%= document_block_counter %>">
-          <h1 class="label">Travel advice</h1>
-          <div class="content">
-            <ol class="document-list">
-              <% @travel_advice.each do |travel_advice| %>
-                <%= content_tag :li, class: 'document-row' do %>
-                  <h2><%= link_to travel_advice["title"], travel_advice["base_path"], class: "govuk-link" %></h2>
-                  <% end %>
-                <% end %>
-            </ol>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: 'Documents',
+          font_size: "l",
+          margin_bottom: 3,
+          padding: true,
+          border_top: 2,
+        } %>
+
+        <%= render(partial: 'shared/document_list_from_rummager', locals: { documents: @publications['results'], type: :publications, documents_count: @publications['total'] }) if @publications['results'].any? %>
+        <%= render(partial: 'shared/document_list_from_rummager', locals: { documents: @consultations['results'], type: :consultations, documents_count: @consultations['total'] }) if @consultations['results'].any? %>
+        <%= render(partial: 'shared/document_list_from_rummager', locals: { documents: @announcements['results'], type: :announcements, documents_count: @announcements['total'] }) if @announcements['results'].any? %>
+
+        <% if @detailed_guides.any? %>
+          <div id="detailed-guidance" class="documents-<%= document_block_counter %>">
```

Comment:
Another mystery ID

Reply:
I think you're right. I got a bit confused thinking the `detailed-guidance` id was used in a test elsewhere but it's not so have now removed (page still looks the same and no test fails)!

As a side note also removed: 
- the `travel-advice` id. I don't think it's needed either and
- `class="documents-<%= document_block_counter %>">` from both elements and
- each containing div since it's also no longer needed